libchisel/src/binaryenopt.rs

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Is this comment needed here? If so, make it use the triple-slash so it can be picked up by rustdoc.